src/saga/vm-console/index.js

```diff
@@ -85,28 +55,22 @@ export function* downloadVmConsole (action) {
       }
 
       data = adjustVVFile({ data, options, usbFilter, isSpice })
-      fileDownload({ data, fileName: 'console.vv', mimeType: 'application/x-virt-viewer' })
+      fileDownload({ data, fileName: `console-${consoleId}.vv`, mimeType: 'application/x-virt-viewer' })
     }
   }
 }
 
-export function* getConsoleOptions (action) {
-  const options = OptionsManager.loadConsoleOptions(action.payload)
-  yield put(setConsoleOptions({ vmId: action.payload.vmId, options }))
-  return options
-}
-
-export function* saveConsoleOptions (action) {
-  OptionsManager.saveConsoleOptions(action.payload)
-  yield getConsoleOptions(getConsoleOptionsAction({ vmId: action.payload.vmId }))
-}
-
+/**
+ * Push a RDP connection file (__console.rdp__) to connect a user to the Windows VM's RDP session
+ */
 export function* getRDPVm (action) {
   const rdpBuilder = new RDPBuilder(action.payload)
   const data = rdpBuilder.buildRDP()
   fileDownload({ data, fileName: 'console.rdp', mimeType: 'application/rdp' })
```

Comment:
Any suggestions for an id to add to this filename?

Reply:
console id just denotes VNC vs SPICE.
So right, no need here. Just the uniqie value, if you feel its needed - as discussed above

Reply:
Yeah, no change.